src/Provider/AbstractProvider.php

```diff
@@ -465,7 +465,8 @@ protected function getAccessTokenRequest(array $params)
         $url     = $this->getAccessTokenUrl($params);
         $options = $this->getAccessTokenOptions($params);
 
-        return $this->getRequest($method, $url, $options);
+        return $this->getRequest($method, $url, $options)
+            ->withHeader('content-type', 'application/x-www-form-urlencoded');
```

Comment:
This isn't quite right... You want to define `$options['headers']` in `getAccessTokenOptions`, right after the `$options['body']` define on line 451.